<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeroView</class>
 <widget class="QWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QLabel" name="heroLabel"/>
   </item>
   <item>
    <widget class="QGraphicsView" name="heroGraphicsView">
     <!--property name="minimumSize">
      <size>
       <width>294</width>
       <height>178</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>294</width>
       <height>178</height>
      </size>
     </property-->
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="bottomPanel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="heroVBoxLayout">
      <item>
       <widget class="QWidget">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel">
           <property name="minimumWidth">
            <number>60</number>
           </property>
           <property name="text">
            <string>Hellfire:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="gameHellfireCheckBox">
           <property name="toolTip">
            <string>Check if it is for a hellfire game</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel">
           <property name="minimumWidth">
            <number>48</number>
           </property>
           <property name="text">
            <string>Multi:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="gameMultiCheckBox">
           <property name="toolTip">
            <string>Check if it is for a multiplayer game</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel">
           <property name="minimumWidth">
            <number>48</number>
           </property>
           <property name="text">
            <string>Speed:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="gameSpeedComboBox">
           <property name="toolTip">
            <string>Game speed</string>
           </property>
           <item>
            <property name="text">
             <string>Normal</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Fast</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Faster</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Fastest</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel">
           <property name="minimumWidth">
            <number>60</number>
           </property>
           <property name="text">
            <string>Difficulty:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="gameDifficultyComboBox">
           <item>
            <property name="text">
             <string>Normal</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Nigthmare</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Hell</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="heroSkillsButton">
           <property name="text">
            <string>Skills</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="heroMonstersButton">
           <property name="text">
            <string>Monsters</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="heroPvPButton">
           <property name="text">
            <string>PvP</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="HeroDetailsWidget" name="mainHeroDetails"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="4"/>
 <resources/>
 <connections/>
 <buttongroups/>
</ui>
